tests/unit/dataactvalidator/test_c8_award_financial.py

```diff
@@ -15,130 +14,127 @@ def test_column_headers(database):
 
 
 def test_equal_fain(database):
-    """Tests that File C (award financial) fain matches
-    File D2 (award financial assistance) fain."""
+    """ Tests that File C (award financial) fain matches File D2 (award financial assistance) fain. """
     tas = _TAS
     af = AwardFinancialFactory(tas=tas, fain='abc', uri=None, allocation_transfer_agency=None)
-    afa = AwardFinancialAssistanceFactory(tas=tas, submisson_id=af.submission_id, fain=af.fain, uri=None)
+    afa = AwardFinancialAssistanceFactory(tas=tas, submisson_id=af.submission_id, fain=af.fain, uri=None,
```

Comment:
Not sure if there is a standard on this: but other tests in the review specifically for D1, would write out the identifier (in this case the fain field), so instead of `af.fain` it would be `abc`